packages/core/README.md

```diff
@@ -18,9 +16,9 @@ This repository contains the static HTML, Javascript, CSS, and images content of
 
 -   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding help, etc.)
 
-## Use a custom domain
+## Manual deployment using custom domain (CNAME)
 
-To use your custom domain, please put it in a file named `CNAME` inside the `scripts` folder of your local clone of deriv-app.
+For manual deployment, it's necessary to use the file named `CNAME`, please create `scripts/CNAME.txt` and put a custom domain to this file. It's only for the deployment on GitHub Pages for your forked repo of deriv-app.
```

Comment:
you need to add more details to this. like the steps we need for deploying deriv-app on custom-link.
maybe something like this: 

1. add a file called `CNAME.txt `in `core/scripts` ( if there is no such directory, create it )
2. put your custom-link inside it. ( Note: The deployment only works on trusted and known domains that end with `binary.sx `and `deriv.com` ) 
3. go to your github and open up your forked deriv-app repository.there you need to go through a couple of steps to create your custom-link
![image](https://user-images.githubusercontent.com/100833613/178926782-14433af7-923d-4d0b-8ff8-bdf7c73b836f.png)
after you finish the above steps you will see this:
![image](https://user-images.githubusercontent.com/100833613/178927020-58db3a74-f06f-44d5-8ab7-5d5e913ebef1.png)
4. from the root directory ( `/deriv-app` ) run the following command:
`npm run deploy:folder .`